MaterialLib/Fluid/Density/LinearTemperatureDependentDensity.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   LinearTemperatureDependentDensity.h
+ * \author: wenqing
+ *
+ * Created on August 10, 2016, 11:34 AM
+ */
+
+#ifndef LINEARTEMPERATUREDEPENDENTDENSITY_H
+#define LINEARTEMPERATUREDEPENDENTDENSITY_H
+
+#include <string>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// Linear temperature dependent density model.
+class LinearTemperatureDependentDensity final : public FluidProperty
+{
+public:
+    /// \param parameters An array contains the five parameters.
+    explicit LinearTemperatureDependentDensity(std::array<double,3>& parameters)
```

Comment:
Documentation different from code: five parameters, but array of length 3. Which one is correct?
Better to document such things describing what the parameters are, not their number, which is visible from the function's signature.